tool/tsh/common/help.go

```diff
@@ -40,10 +40,10 @@ EXAMPLES:
   For passwordless authentication use:
   $ tsh login --auth=passwordless`
 
-	// missingPrincipalsFooter is printed at the bottom of `tsh ls` when no results are returned.
-	missingPrincipalsFooter = `
-  Not seeing nodes? Your user may be missing Linux principals. If trying Teleport for the first time, follow this guide:
+	// emptyNodesFooter is printed at the bottom of `tsh ls` when no results are returned for nodes.
+	emptyNodesFooter = `
+  Not seeing nodes? Either no nodes are available or roles for your user do not match to node labels.
```

Comment:
```suggestion
  Not seeing nodes? Either no nodes are available or your user's roles do not match the labels of at least one node.
```
I thought it would make it a bit clearer to quantify this.